apps/server/src/modules/learnroom/service/group-deleted-handler.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Group, GroupDeletedEvent } from '@modules/group';
+import { Injectable } from '@nestjs/common';
+import { EventsHandler, IEventHandler } from '@nestjs/cqrs';
+import { Course } from '../domain';
+import { CourseDoService } from './course-do.service';
+
+@Injectable()
+@EventsHandler(GroupDeletedEvent)
+export class GroupDeletedHandlerService implements IEventHandler<GroupDeletedEvent> {
+	constructor(private readonly courseService: CourseDoService) {}
+
+	public async handle(event: GroupDeletedEvent): Promise<void> {
+		await this.removeCourseSyncReference(event.target);
+	}
+
+	private async removeCourseSyncReference(group: Group): Promise<void> {
+		const courses: Course[] = await this.courseService.findBySyncedGroup(group);
```

Comment:
Does cqrs logs something? Otherwise i would suggets to add loggables for publishing and handling events.. without those bug hunting will be really hard. 
This was one reason why i had a wrapper for the nestjs event bus

Reply:
@alweber-cap Which log level do you prefer? Will be info logged in prod then i would use info..